static/explorer.css

```diff
@@ -451,6 +451,10 @@ button.dropdown-item.btn.btn-light.btn-sm {
 
 .navbar-nav a.nav-link {
     font-size: 14px;
+    border: rgb(140, 140, 140) 1px solid;
```

Comment:
Not sure how I feel about the border here, but the most important part:
Can you move the border colour styles to `./static/themes/*-theme.css`? It will make it easier to override it on the dark theme. (Settings > Theme > Dark) Can you make sure it also looks ok for the dark theme? :)

Reply:
Moved them into the  theme types but kept the same values, and left the styling not related to color in the primary theme.